tests/add_sqlite_columns.py

```diff
@@ -0,0 +1,36 @@
+# Take the directory name of argv[1] 
+# Recursively walk the directory, find all SQLite databases
+# In each database, check that there is a propmt_id column
+# Add columns llm_info TEXT and cost REAL 
+# The default value for llm_info is "{}"
+
+import os
+import sqlite3
+import sys
+
+_llm_calls = "LLMCalls"
+
+def add_columns_to_db(db_path):
+    conn = sqlite3.connect(db_path)
+    c = conn.cursor()
+    c.execute(f"PRAGMA table_info{_llm_calls}")
+    columns = [row[1] for row in c.fetchall()]
+    if "llm_info" not in columns:
+        c.execute(f"ALTER TABLE {_llm_calls} ADD COLUMN llm_info TEXT DEFAULT '{{}}'")
```

Comment:
let's make the column name configurable from the args of the script. This way we can reuse it next time.